src/features/student/views/StudentToolsTracesView/components/StudentToolsTracesViewNotice/StudentToolsTracesViewNotice.vue

```diff
@@ -1,11 +1,15 @@
 <script setup lang="ts">
-import { useTracesConfigurationQuery, useUnassignedTracesSummaryQuery } from '@/features/student/queries'
+import type { TracesSummaryDTO } from '@/api/avenir-esr'
+import { useTracesConfigurationQuery } from '@/features/student/queries'
 import { AvNotice } from '@avenirs-esr/avenirs-dsav'
 import { useI18n } from 'vue-i18n'
 
+const { unassignedTracesSummary } = defineProps<{
+  unassignedTracesSummary: TracesSummaryDTO | undefined
+}>()
```

Comment:
Pas besoin de destruct vu que c'est utilisé que dans la template

```suggestion
defineProps<{
  unassignedTracesSummary: TracesSummaryDTO | undefined
}>()
```